label_options/usaid-laos-ev-label-options.json

```diff
@@ -0,0 +1,124 @@
+{
+    "MODE": [
+      {"value":"walk", "baseMode":"WALKING", "met_equivalent":"WALKING", "kgCo2PerKm": 0},
+      {"value":"e-tuk_tuk", "baseMode":"MOPED", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.085416859},
+      {"value":"tuk_tuk", "baseMode":"MOPED", "met_equivalent":"IN_VEHICLE", "kgCo2PerKm": 0.231943784},
```

Comment:
we should think about the internal representation of the tuk_tuk - the translation can obviously still be "ເດີນທາດ້ວຍ ລົດຕຸກຕຸກ ຫລື ລົດສາມລໍ້" in Lao. Wikipedia lists "auto-rickshaw" as the primary name
https://en.wikipedia.org/wiki/Auto_rickshaw
with multiple translations :)

I would suggest maybe using auto_rickshaw as the internal mode for consistency with wikipedia.

Also, I don't think we can use MOPED as the baseMode - I don't think it is supported in the UI. It would have to be "CAR" since that is what we use for taxis and to my knowledge, there are very few privately owned autos, at least in India. In fact, when somebody bought a private auto instead of a car, it made the newspaper :)

We do support a _MODE_ of MOPED - but I don't think it should be a _base_ mode since it is typically not auto-inferred.

I see that there is a base_mode map that contains all the mappings, including the user labeled modes such as E_CAR and MOPED
https://github.com/e-mission/e-mission-phone/blob/cc0e41935f76e098c06d1c955c720cf6e975a66f/www/js/diary/diaryHelper.ts#L43

Couple of questions around that:
- I thought that the base modes were the corresponding "detected" modes. I don't think we should add a new basemode that is an exact match for every mode in the dropdown
- why do we need to add MOPED to the phone code? I see 

https://github.com/e-mission/e-mission-phone/commit/eadc652818efdf6325cf937bf4eb828dffee7b9a

but requiring code changes for every set of dynamic labels defeats the purpose of having the dynamic labels in the first place.

@JGreenlee can you clarify?

Reply:
> * I don't think we should add a new basemode that is an exact match for every mode in the dropdown
> but requiring code changes for every set of dynamic labels defeats the purpose of having the dynamic labels in the first place.

To add a set of dynamic labels, we *don't* require code changes, so long as there is a "close enough" base mode for it. Not every rich mode warrants a different icon -- for example, "drove alone" and "shared ride" share the same icon just fine.

But wouldn't it be kind of off-putting if we had to resort to a bicycle icon for moped? And wouldn't it be helpful to users to have e-cars be distinguishable from gas cars?
The only concrete limitation we have is the icon library, and the icon library we are using has icons specifically for electric car and moped. So why not use them?

Reply:
>I would suggest maybe using auto_rickshaw as the internal mode for consistency with wikipedia.

done in [81135dd](https://github.com/e-mission/nrel-openpath-deploy-configs/pull/39/commits/81135dd93f485b315eac5b588a1c301d0042ee1c)

Reply:
> we don't require code changes, so long as there is a "close enough" base mode for the new rich mode. 

That's good. I want to make sure that config changes can be made almost instantaneously and don't have to wait for a code release, with all the overhead that it involves.

> But wouldn't it be kind of off-putting if we had to resort to a bicycle icon for moped? And wouldn't it be helpful to users to have e-cars be distinguishable from gas cars?
> The only concrete limitation we have is the icon library, and the icon library we are using has icons specifically for electric car and moped. So why not use them?

I thought that the base modes were the modes detected by the sensing algorithms, since that is how we use it in the "MobilityNet confusion matrix" paper. I guess this definition is basically "modes that have icons". In which case, this is fine.

Reply:
Ah, sorry for the confusion.

> the modes detected by the sensing algorithms

In the phone code, I've been referring to these as `MotionTypes` since that is what they are called in the server code.